JOSESwift/Sources/JWEHeader.swift

```diff
@@ -211,4 +211,14 @@ extension JWEHeader: CommonHeaderParameterSpace {
             return parameters["crit"] as? [String]
         }
     }
+
+    /// The zip header parameter indicates the content has been compressed before encryption
+    public var zip: String? {
```

Comment:
Do you think it would make sense to return a `CompressionAlgorithm` instead of a `String` here? It would enable easier switching on the value for the user (without having to convert the String to an enum case or doing string comparison).